docs/src/pages/system/sx/sx.md

```diff
@@ -0,0 +1,92 @@
+# The `sx` prop
+
+<p class="description">Utility for styling elements inline, using the theme values.</p>
+
+## Getting Started
+
+Each `@material-ui/core` component supports the `sx` prop. The prop is a superset of CSS, that let's you add any CSS to the underlaying component, while mapping values to different theme keys. This should help you to keep your application look consistent.
+
+We recommend you use this prop whenever you need to add one-of style overrides on the Material-UI components. If you repetedly add the same styles on some component, than the `styled()` is better alternative, as it allows you to specify the overrides only ones and reuse them in all component instances.
```

Comment:
I'll work next on the `styled()` utility docs page, will link it here once it's done